### PR TITLE
modify IP setting for workflow as per new change

### PR DESCRIPTION
**Pull request by masco-enovance**

### CODE REVIEW

**Comment by masco-enovance** [comment]
run skydive-go-fmt

**Comment by masco-enovance** [comment]
run skydive-unit-tests
run skydive-functional-tests-backend-orientdb
run skydive-functional-tests-backend-elasticsearch
run skydive-compile-tests
run skydive-cdd-overview-tests

**Comment by masco-enovance** [comment]
run skydive-cdd-overview-tests
run skydive-functional-tests-backend-orientdb

File: statics/js/components/inject-form.js [hunk 1]
```diff
@@ -217,16 +217,19 @@ Vue.component('inject-form', {
...
+      } else if (ipFamily == "6") {
+        if (md.Neutron && md.Neutron.IPV6) return md.Neutron.IPV6[0];
+        if (md.IPV6) return md.IPV6[0];
       } else {
         return "";
       }
     },
 
     getMAC: function(node) {
+      if (node.metadata.ExtID && node.metadata.ExtID.attached-mac) return node.metadata.ExtID.attached-mac;
```
Comment (hunchback) [inline statics/js/components/inject-form.js]: You bracket notation (as attribute name contains a hyphen):

```
node.metadata.ExtID["attached-mac"]
```

Else hyphen is interpreted as a minus

Comment (masco-enovance) [reply statics/js/components/inject-form.js]: Thanks for the review Aidan, Fixed!

**Comment by masco-enovance** [comment]
run skydive-functional-tests-backend-elasticsearch
run skydive-functional-tests-backend-orientdb

File: statics/js/components/inject-form.js [hunk 2]
```diff
@@ -217,16 +217,19 @@ Vue.component('inject-form', {
     getIP: function(node) {
       md = node.metadata;
```
Comment (hunchback) [inline statics/js/components/inject-form.js]: if after: 

```
md = node.metadata;
```

you append:
```
md = md.Neutron ? md.Neutron : md;
```

and then the rest if the code is left un modified

Comment (masco-enovance) [reply statics/js/components/inject-form.js]: done.

**Comment by masco-enovance** [comment]
run skydive-functional-tests-backend-orientdb

**Pull request merged**